### PR TITLE
Add 'last_updated' field to ContentMetadataCache

### PR DESCRIPTION
**Pull request by jonboiser**
Implements #1496.

1. Add a `last_updated` `DateTimeField` to `ContentMetadataCache`
1. `last_updated` field is set to current datetime when channel is imported for the first time from remote source, local USB drive, or from the device's `./kolibri/content` folder upon first startup.

TODO
1. ~~Implement logic for filling in updated value for different cases~~

### CODE REVIEW

File: kolibri/content/models.py [hunk 1]
```diff
@@ -302,5 +303,7 @@ class ChannelMetadataCache(ChannelMetadataAbstractBase):
     This class stores the channel metadata cached/denormed into the primary database.
     """
 
+    last_updated = models.DateTimeField(null=True)
```
Comment (rtibbles) [inline kolibri/content/models.py]: I just merged my PR for `DateTimeTZField`, after a brief discussion with @indirectlylit and @jamalex seems like a good idea to use this in most places.

`from kolibri.core.fields import DateTimeTZField`

File: kolibri/content/utils/annotation.py [hunk 2]
```diff
@@ -7,15 +8,27 @@
...
+        ch_metadata_obj, _ = ChannelMetadataCache.objects.update_or_create(
+            id=db_name,
+            defaults=update_values,
+        )
+
+        # Records a new last_updated only if channel is brand new. Does not
+        # handle case where channel's version is upgraded, which is not
+        # yet supported on UI anyway
+        if ch_metadata_obj.last_updated is None:
+            ch_metadata_obj.last_updated = timezone.now()
```
Comment (rtibbles) [inline kolibri/content/utils/annotation.py]: Can use `from kolibri.utils.time import local_now` here to get the time in the current system timezone (paired with the above field change).

File: kolibri/content/migrations/0002_channelmetadatacache_last_updated.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2017-06-05 20:54
+from __future__ import unicode_literals
```
Comment (rtibbles) [inline kolibri/content/migrations/0002_channelmetadatacache_last_updated.py]: Can just delete and recreate this migration with the different field.

**Comment by jonboiser** [comment]
@rtibbles 

1. Changed the `DateTimeField` to `DateTimeTzField`
1. Used `local_now` utility

**Pull request merged**